src/Element.cc

```diff
@@ -535,11 +539,20 @@ void Element::PrintValuesImpl(const std::string &_prefix,
       {
         if ((*eiter)->GetExplicitlySetInFile() || _includeDefaultElements)
         {
-          (*eiter)->ToString(_prefix + "  ",
-                             _includeDefaultElements,
-                             _includeDefaultAttributes,
-                             _config,
-                             _out);
+          if (_config.PreserveIncludes()
+              && (*eiter)->GetIncludeElement() != nullptr)
+          {
+            _out << (*eiter)->GetIncludeElement()
+                            ->ToString(_prefix + "  ", _config);
+          }
```

Comment:
Do you know if this extra check is necessary? I think we can just let the following `ToString` call happen and it will recursively call `PrintValuesImpl` which has these checks at the beginning of the function.

Reply:
No it was not necessary and something I forgot to remove, thank you for catching that. I also removed the extra `if ((*eiter)->GetExplicitlySetInFile() || _includeDefaultElements)` check because it didn't seem to be needed either and all tests passed locally. Let me know if you think it should be put back 8b5bd672b50e35f6658619b737c2292a36c79499